tests/unit/test_parameter_sets.py

```diff
@@ -136,3 +137,16 @@ def test_set_formation_concentrations(self):
         assert (
             parameter_set["Initial concentration in positive electrode [mol.m-3]"] > 0
         )
+
+    @pytest.mark.unit
+    def test_evaluate_symbol(self):
+        parameter_set = pybop.ParameterSet.pybamm("Chen2020")
+
+        for param in [
+            1.0,
+            Scalar(0),
+            Parameter("Positive electrode porosity"),
```

Comment:
```suggestion
            Parameter("Positive electrode porosity"),
            1.0 - FunctionParameter("Positive electrode porosity")
```
so that you cover all the lines